payjoin/tests/integration.rs

```diff
@@ -740,7 +810,7 @@ mod integration {
         outputs.insert(pj_uri.address.to_string(), pj_uri.amount.unwrap_or(Amount::ONE_BTC));
         let options = bitcoincore_rpc::json::WalletCreateFundedPsbtOptions {
             lock_unspent: Some(true),
-            fee_rate: Some(Amount::from_sat(2000)),
+            fee_rate: Some(Amount::from_sat(1000)),
```

Comment:
Changing this to the minimum relay feerate ensures that tests fail if the receiver would add inputs/outputs that cannot be covered by the sender's additional fee contributions.

Note that in the `v2_to_v2` test the sender cannot cover additional fees because they don't have a change output. The current API doesn't let the receiver pay for additional fees (tx-cut through fixes this). So for now, in `build_sweep_psbt` we keep the fee_rate at 2000 to ensure the sender overpays and the resulting tx passes min fee relay checks (see also [L341](https://github.com/payjoin/rust-payjoin/pull/351/files#diff-6b73b9a6c1b1737449037f23519b7340a69b3fd31f7444c4a8873e07bc5e70d1R341)).

Reply:
I was a bit confused by the units, but it seems like `bitcoin::policy::DEFAULT_MIN_RELAY_FEE` can be used here since it's expressed in sat/kvB and the `feeRate` argument to WalletCreateFundedPsbtOptions is serialized in BTC/kvB, so the units work out.

I think using the const is a bit more meaningful so I added a commit directly to this

Reply:
Yeah, I was also confused/frustrated by the fact that the `fee_rate` param is expressed as an Amount and not a FeeRate. Thanks for making that change